tests/dummy/app/templates/docs/links.md

```diff
@@ -1,4 +1,4 @@
-# Links
+# Linking Between Engines (and Applications)
```

Comment:
I think this should be explained in the doc itself, the idea is that this doc is about the link-to-external component, maybe we call it link-to-external instead of links? That actually might make this more obvious.

Reply:
I don't think so, to be honest this page is not about link components, but also about `externalRoutes`. We should refactor this page and create another section about `Linking Between Engines and Apps` grabbing `externalRoutes` such an API for links